test/testdefs.jl

```diff
@@ -9,6 +30,7 @@ function runtests(name, isolate=true)
             # code it in the test
             mod_name = Symbol("Test", rand(1:100), "Main_", replace(name, '/', '_'))
             m = @eval(Main, module $mod_name end)
+            eval(m, :(import Main: test_exename, test_exeflags, addprocs_with_testenv))
```

Comment:
CodeCoverage wraps `testdefs.jl` in a module `BaseTestRunner`. Importing `test_exename, test_exeflags, addprocs_with_testenv` into Main after this line - https://github.com/JuliaCI/CoverageBase.jl/blob/275bb9fb93298e9b48471b8a519acb30315c79c0/src/CoverageBase.jl#L28 - is the correct fix ?